examples/pico/car/pom.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (c) 2023 Oracle and/or its affiliates.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <groupId>io.helidon.examples.pico</groupId>
+        <artifactId>helidon-examples-pico-project</artifactId>
+        <version>4.0.0-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>io.helidon.examples.pico.car</groupId>
+    <artifactId>helidon-examples-pico-car-project</artifactId>
+    <name>Helidon Pico Examples - Car</name>
+
+    <packaging>pom</packaging>
+
+    <properties>
+<!--        <version.lib.lombok>1.18.24</version.lib.lombok>-->
+        <version.lib.dagger>2.43</version.lib.dagger>
+
+        <!--        <checkstyle.skip>true</checkstyle.skip>-->
```

Comment:
Are these two commented lines leftovers?

Reply:
will fix in next push.

Reply:
@barchetta is it ok to have the dagger2 dependency down at this level or somewhere higher up?

Reply:
For an example this is fine as we don't want this dependency polluting anything else.